content/en/tracing/setup_overview/proxy_setup/_index.md

```diff
@@ -53,66 +53,66 @@ Datadog APM is included in Envoy v1.9.0 and newer.
-
-```yaml
-  clusters:
-  ... existing cluster configs ...
-  - name: datadog_agent
-    connect_timeout: 1s
-    type: strict_dns
-    lb_policy: round_robin
-    load_assignment:
-      cluster_name: datadog_agent
-      endpoints:
-      - lb_endpoints:
-        - endpoint:
-            address:
-              socket_address:
-                address: localhost
-                port_value: 8126
-```
-
-The `address` value may need to be changed if Envoy is running in a container or orchestrated environment.
-
-Envoy's tracing configuration needs to use the Datadog APM extension.
-
-```yaml
-tracing:
-  http:
-    name: envoy.tracers.datadog
-    typed_config:
-      "@type": type.googleapis.com/envoy.config.trace.v2.DatadogConfig
-      collector_cluster: datadog_agent   # matched against the named cluster
-      service_name: envoy-example        # user-defined service name
-```
+1. Add a cluster for submitting traces to the Datadog Agent:
+
+   ```yaml
+    clusters:
+    ... existing cluster configs ...
+    - name: datadog_agent
+      connect_timeout: 1s
+      type: strict_dns
+      lb_policy: round_robin
+      load_assignment:
+        cluster_name: datadog_agent
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address:
+                  address: dd-agent
+                  port_value: 8126
+   ```
+
+   Change the `address` value if Envoy is running in a container or orchestrated environment.
+
+2. Set Envoy's tracing configuration to use the Datadog APM extension:
+
+   ```yaml
+    tracing:
+      provider:
+        name: envoy.tracers.datadog
+        typed_config:
+          "@type": type.googleapis.com/envoy.config.trace.v3.DatadogConfig
+          collector_cluster: datadog_agent # matched against the named cluster
+          service_name: envoy-v1.19        # user-defined service name
+   ```
 
-The `collector_cluster` value must match the name provided for the Datadog Agent cluster.
-The `service_name` can be changed to a meaningful value for your usage of Envoy.
+   The `collector_cluster` value must match the name provided for the Datadog Agent cluster. The `service_name` can be changed to a meaningful value for your usage of Envoy.
 
-Finally, the `http_connection_manager` sections need to include additional configuration to enable tracing.
+3. Include the following additional configuration in the `http_connection_manager` sections to enable tracing:
 
-```yaml
-      - name: envoy.http_connection_manager
-        typed_config:
-          "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
-          tracing: {}
-```
+   ```yaml
+    - name: envoy.filters.network.http_connection_manager
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+        generate_request_id: true
+        tracing: {}
```

Comment:
This section now needs configuring with the tracing provider and other details (same as the full example further down).